docs/config.md

```diff
@@ -0,0 +1,113 @@
+# Configuring the HYPR Authenticator
+To use the HYPR authenticator with WSO2 Identity Server, first you need to configure  the authenticator with
+WSO2 Identity Server. See the instructions given below on how to configure the HYPR authenticator with
+WSO2 Identity Server using a sample app.
+
+To test the approach, first an end-user account should be registered in the application created via the 
+HYPR Control Center and the end user should have the HYPR mobile application installed in the smartphone, as well as 
+the smartphone registered with the end user account.
+
+After deploying the HYPR Authenticator to WSO2 IS, the Authenticator can be configured from the
+WSO2 IS Console.
+
+## Prerequisites
+To use the connector, you'll need:
+
+- A configured HYPR environment.
+- A HYPR user account and mobile app to use for testing.
+
+Note: Get the support from HYPR to configure a HYPR Application via the HYPR Control Center.
+
+## Setting up and installing the HYPR connector
+
+**Step 1:** Extracting the project artifacts
+1. Clone the `identity-outbound-auth-hypr` repository.
+2. Build the project by running ```mvn clean install``` in the root directory.
+
+Note : The latest project artifacts can also be downloaded from the Connector Store.
+
+**Step 2:** Deploying the HYPR Authenticator
+
+1. Navigate to `identity-outbound-auth-hypr/components` → `org.wso2.carbon.identity.application.authenticator.hypr` 
+→ `target`.
+2. Copy the `org.wso2.carbon.identity.application.authenticator.hypr-1.0.0-SNAPSHOT.jar` file.
+3. Navigate to `<IS_HOME>/repository/components/dropins`.
+4. Paste the `.jar` file into the dropins directory.
+5. Alternatively it's possible to drag and drop the `.jar` file to the dropins directory.
+6. Next navigate to `identity-outbound-auth-hypr/components` → 
+`org.wso2.carbon.identity.application.authenticator.hypr.common` → `target`.
+7. Copy the `org.wso2.carbon.identity.application.authenticator.hypr.common-1.0.0-SNAPSHOT.jar` file.
+8. Navigate to `<IS_HOME>/repository/components/lib` directory and paste the `.jar` file.
+
+**Step 3:** Deploying the HYPR REST API
+1. Navigate to `identity-outbound-auth-hypr/components` → `org.wso2.carbon.identity.application.authenticator.hypr.rest` 
+→`org.wso2.carbon.identity.application.authenticator.hypr.rest.dispatcher`→ `target`.
+2. Copy the `api#hypr.war` file.
+3. Navigate to `<IS_HOME>/repository/deployment/server/webapps`.
+4. Paste the `.war` file into the webapps directory.
+5. Next navigate to `<IS_HOME>/repository/conf`.
+6. Open `deployment.toml` file.
+7. Add the following lines of codes.
+```toml
```

Comment:
should be formatted to render under step 7

Reply:
Resolves via [ab5e02d](https://github.com/wso2-extensions/identity-outbound-auth-hypr/pull/2/commits/ab5e02d88c973fd971f23c3253533a26e6bbf16b)